bpf/lib/nodeport.h

```diff
@@ -865,6 +865,7 @@ nodeport_rev_dnat_ingress_ipv6(struct __ctx_buff *ctx, struct trace_ctx *trace,
 	struct ipv6hdr *ip6;
 	__u32 tunnel_endpoint __maybe_unused = 0;
 	__u32 dst_sec_identity __maybe_unused = 0;
+	__u32 src_sec_identity __maybe_unused;
```

Comment:
nit: Wondering if this should be something like this which can always specify some security identity (could help reduce cognitive burden on developers reading this function and the other function with similar logic by converting 15 lines of very specific identity allocation logic into a one-liner initialization that contains the conditional initialization logic below).
```suggestion
	__u32 src_sec_identity __maybe_unused = initialize_sec_identity();
```

Reply:
I'd prefer *not* do that :slightly_smiling_face:. I don't consider the list of security identities to be well thought-through, and would want to avoid any kind of impression that this is a sane selection of defaults. Or that it should be re-used anywhere else. We make it disappear by reasoning about what the *correct* behavior is in each of the programs.

Reply:
Having a single declaration for use across IPv4/IPv6 with an explicit comment warning the reader that the contents are subject to change might help to communicate that more clearly ;) But I defer to you on this.

Reply:
@ti-mo took care of this in a different way now :)